pytorch_lightning/utilities/__init__.py

```diff
@@ -19,3 +20,8 @@
 FLOAT16_EPSILON = numpy.finfo(numpy.float16).eps
 FLOAT32_EPSILON = numpy.finfo(numpy.float32).eps
 FLOAT64_EPSILON = numpy.finfo(numpy.float64).eps
+
+
+class AMPType(Enum):
+    APEX = 'apex'
+    NATIVE = 'native'
```

Comment:
I didn't find the place where you convert the string trainer arg to this type, where is it?

Reply:
It is in `init_amp` in AMP mixin...